src/App.tsx

```diff
@@ -1,6 +1,43 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((goods1, goods2) => {
```

Comment:
lets use better naming
```suggestion
  visibleGoods.sort((firstGood, secondGood) => {
```